### ISSUE TITLE
No conversion to symbol

### ISSUE THREAD

**Issue by TimoKramer** [clj-kondo/clj-kondo#1662]
**version**

clj-kondo v2022.04.08

**platform**

native on Linux MNZNB0441 5.10.16.3-microsoft-standard-WSL2 #1 SMP Fri Apr 2 22:23:49 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

**problem**

My config was responsible for an error that is not easy to understand. Was searching for related stuff and found something on paid slack (yeah). This seems loosely related: https://github.com/clj-kondo/clj-kondo/issues/640

I am getting:
```
Unexpected error. Please report an issue.
java.lang.IllegalArgumentException: no conversion to symbol
```

**repro**

`CLJ_KONDO_DEV=true clj-kondo --lint foo.cljs --config '{:output {:format :json}}' --debug`

**config**

```
{:linters
  {:unresolved-symbol
    {:exclude [(com.mycompany.dp.monitor.styles)]}}}
```

**expected behavior**

no errors

[foo.cljs.txt](https://github.com/clj-kondo/clj-kondo/files/8531436/foo.cljs.txt)

Thanks for the good work!

**Comment by borkdude** [comment]
Thanks!

**Comment by borkdude** [comment]
If it wasn't clear yet, the issue is that the symbol should be fully qualified.

**Comment by borkdude** [comment]
Repro:

```
$ clj-kondo --config '{:linters
  {:unresolved-symbol
    {:exclude [(com.mycompany.dp.monitor.styles)]}}}' --lint - <<< 'x'
```
